.github/workflows/.github/workflows/webext-lint.yml

```
@@ -0,0 +1,23 @@
+name: Web-Ext lint
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

```suggestion
  push:
    branches: [ "master" ]
  pull_request:
    branches: [ "master" ]
```

Ah sorry your branch name is still "outdated" :wink: 
Fixing it…